NMS-14735: Beyond Quick Start (#5513)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/user-management/user-groups.adoc

Sending notifications to groups lets you build more complex notification workflows, and makes it easier to manage recipients.

[[ga-user-group-create]]
== Create a group

To create a group of users, follow these steps:

. Log in as a user with admin permissions.
. Click the *gear* symbol at the top-right of the screen.
. Under OpenNMS System, click menu:Configure Users, Groups and On-Call Roles[Configure Groups].
. Click *Add New Group* and specify the its name and description.
+
WARNING: You cannot use angle brackets (`<>`), single (`'`) and double (`"`) quotation marks, or the ampersand symbol (`&`) in a group name.

. Add a user to the group by selecting them from the Available Users list and clicking *>>* to move them to the Currently in Group list.
. (Optional) Assign responsibility categories (Routers, Switches, Servers, and so on).
. (Optional) Create a <<deep-dive/user-management/user-config.adoc#ga-user-schedule, duty schedule>>.
. Click *Finish* to apply the changes.

NOTE: The order of users in a group is important.
Notifications are sent to users in order from the top of the group membership list to the bottom.

=== Modify group membership

> Users may need to be added or removed from groups as team membership changes.

Reviewer: ```suggestion
You may need to add or remove users from groups as team membership changes.
```
